rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/450_ip_prefix.yml

```diff
@@ -22,25 +23,25 @@ setup:
         refresh: true
         body:
           - { "index": { } }
-          - { "ipv4": "192.168.1.10", "ipv6": "2001:db8:a4f8:112a:6001:0:12:7f10", "value": 10, ip: "192.168.1.10" }
+          - { "ipv4": "192.168.1.10", "ipv6": "2001:db8:a4f8:112a:6001:0:12:7f10", "value": 10, "ip": "192.168.1.10" }
```

Comment:
I added double quotes here around the field name just for consistency.

Reply:
Sure